### PR TITLE
Avoid running untrusted code as root to install nodejs

### PR DESCRIPTION
**Pull request by kalvdans**
Instead follow the recommended instructions.

### CODE REVIEW

File: site/content/contribute/more-info/desktop/developer-setup/ubuntu.md [hunk 1]
```diff
@@ -1,8 +1,13 @@
 1. Open Terminal
-2. Install NodeJS from NodeSource:
+2. Install NodeJS from [NodeSource](https://github.com/nodesource/distributions#installation-instructions)
```
Comment (kalvdans) [inline site/content/contribute/more-info/desktop/developer-setup/ubuntu.md:2]:
```suggestion
2. Install NodeJS from [NodeSource](https://github.com/nodesource/distributions#installation-instructions):
```

**Review by cwarnermm — APPROVED** [review]
Approved assuming that the changes proposed are also approved by Engineering.

**Comment by devinbinnie** [comment]
@kalvdans Thanks for the PR! Looking at this now, I'm thinking it might make more sense to instruct the user to install Node via NVM. I think it's a bit easier to do. Would you be interested in adding a step to install NVM and the latest Node LTS?

**Comment by devinbinnie** [comment]
@kalvdans Are you still interested in finishing this up?

**Comment by kalvdans** [comment]
> @kalvdans Thanks for the PR! Looking at this now, I'm thinking it might make more sense to instruct the user to install Node via NVM. I think it's a bit easier to do. Would you be interested in adding a step to install NVM and the latest Node LTS?

@devinbinnie Sorry I missed your comments the first time, thanks for the reminder! I rather do version control with apt/dpkg which is native to Ubuntu as long as nodejs maintains their ubuntu repository. I'm not a web programmer really so I'm happy to merge this PR and let someone more passionated about nvm make another PR.

**Comment by devinbinnie** [comment]
> > @kalvdans Thanks for the PR! Looking at this now, I'm thinking it might make more sense to instruct the user to install Node via NVM. I think it's a bit easier to do. Would you be interested in adding a step to install NVM and the latest Node LTS?
> 
> @devinbinnie Sorry I missed your comments the first time, thanks for the reminder! I rather do version control with apt/dpkg which is native to Ubuntu as long as nodejs maintains their ubuntu repository. I'm not a web programmer really so I'm happy to merge this PR and let someone more passionated about nvm make another PR.

Sure, I'm happy to merge as-is.

**Review by devinbinnie — APPROVED** [review]
Thanks @kalvdans!

**Pull request merged**